### DISCUSSION TITLE
LibreSBC installation

### DISCUSSION THREAD

**Discussion by aequser** [hnimminh/libresbc#52] [Q&A] [answered]
Hello, you help will be very appreciate.

I'm trying to install libre sbc and show me a role error:

ERROR! the role 'platform' was not found in /root/libresbc/build/ansible/playbooks/roles:/root/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/root/libresbc/build/ansible/playbooks

What I making wrong?

Thanks in advance.

**Comment by hnimminh** [comment, marked as answer]
look like the ansible does not found the role. 
did you follow step in https://github.com/hnimminh/libresbc/blob/master/documents/Installation.md
